### ISSUE TITLE
Support of preprocessor keywords

### ISSUE THREAD

**Issue by langovoi** [cappuccino/acorn-objj#1]
Did you plan to add support of #pragma, #define, #include and etc?

**Comment by mrcarlberg** [comment]
Hi!
This parser does not support preprocessing. If you are interested in preprocessing please use https://github.com/mrcarlberg/objj-parser

**Comment by langovoi** [comment]
@mrcarlberg maybe you're planning to add this in the future?
I find this parser is better, because it not modify acorn.

**Comment by mrcarlberg** [comment]
Yes it is true that this parser is better as it is up to date with the current acorn parser. But the guy who wrote this decided to not implement a preprocessor. It is not finished and there are currently no-one working on this.

The other parser is the one currently used in the Cappuccino project (http://www.cappuccino-project.org).

Please feel free to create a preprocessor if you want to contribute 😄
